### PR TITLE
DDS Router library pull request release notes [14080]

### PR DESCRIPTION
**Pull request by rsanchez15**
Merge after:
- #162
- #161

### CODE REVIEW

File: docs/rst/notes/forthcoming_version.rst [hunk 1]
```diff
@@ -1,7 +1,41 @@
...
+  - `ddsrouter_utils`:  C++ library which implements various utility functions.
+  - `ddsrouter_core`: C++ library which implements the DDS Router operation and exports the DDS Router C++ API.
+  - `ddsrouter_yaml`:  C++ library to parse the DDS Router *yaml* configuration files.
+  - `ddsrouter_tool`: DDS Router end-user application.
+
+* Build internal topic tracks via the `builtin-topics` configuration.
+
+This release includes the following **Routing features**:
+
+* Apply DDS Topic filter rules using the `allowlist` and `blocklist` lists.
```
Comment (jparisu) [inline docs/rst/notes/forthcoming_version.rst:19]: Dynamic module is not included in this release. I do not understand what do you mean here.

File: docs/rst/notes/forthcoming_version.rst [hunk 2]
```diff
@@ -1,7 +1,41 @@
...
+
+Next release includes the following **User Interface features**:
+
+* Allow using wildcard characters to define topics in `allowlist` and `blocklist`.
```
Comment (jparisu) [inline docs/rst/notes/forthcoming_version.rst:23]: Again, I'll add this in dynamic PR

**Review by jparisu — CHANGES_REQUESTED** [review]

File: docs/rst/notes/forthcoming_version.rst (hunk 1 above)
Comment (rsanchez15) [reply docs/rst/notes/forthcoming_version.rst:19]: Include in PR #154

File: docs/rst/notes/forthcoming_version.rst (hunk 2 above)
Comment (rsanchez15) [reply docs/rst/notes/forthcoming_version.rst:23]: Include in PR https://github.com/eProsima/DDS-Router/pull/154

**Review by jparisu — APPROVED** [review]
LGTM

**Pull request merged**